dashboard/test/lib/pd/survey_pipeline/retriever_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'test_helper'
+require 'pd/survey_pipeline/retriever.rb'
+
+module Pd::SurveyPipeline
+  class DailySurveyRetrieverTest < ActiveSupport::TestCase
+    include Pd::WorkshopConstants
+
+    setup_all do
+      @form_ids = [11_000_000_000_000, 11_000_000_000_001]
```

Comment:
fwiw, this linter requirement bugged me enough that I'm currently thinking of going with:
```
["90525028928158"].map(&:to_i)
```
in my own code.

Reply:
I'm not a fan either, but I'm getting better at ignoring it. For the context, I use the same number length as real JotForm id so it stands out in debugging log.